os/kernel/init/os_bringup.c

```diff
@@ -272,6 +272,10 @@ static inline void os_do_appstart(void)
 	task_manager_drv_register();
 #endif
 
+#ifdef CONFIG_MESSAGING_IPC
+	messaging_initialize();
```

Comment:
I think this is not related with os_bringup. How about moving this to preapp init?

Reply:
The preapp_init is in user space so that you should not move there.